server/controllers/finance/generalLedger/index.js

```diff
@@ -42,8 +46,46 @@ function list(req, res, next) {
     `;
 
   db.exec(sql)
-    .then(function (rows) {
-        res.status(200).json(rows);
+    .then((rows) => {
+      res.status(200).json(rows);
     })
     .catch(next);
 }
+
+/**
+ * GET /general_ledger/accounts
+ * list accounts and their solds
+ */
+function listAccounts(req, res, next) {
+  getlistAccounts()
+  .then((rows) => {
+    res.status(200).json(rows);
+  })
+  .catch(next)
+  .done();
+}
+
+/**
+ * @function getlistAccounts
+ * get list of accounts
+ */
+function getlistAccounts() {
+  const sql =
+    `SELECT aggregator.id, aggregator.number, aggregator.label,
+      IF(aggregator.balance >= 0, aggregator.balance, 0) AS debtor_sold,
```

Comment:
In my opinion, it is better to use `balance` instead of `sold`, to keep it in the same language.  This won't block this PR though.